EightfoldAI/octuple
# feat: Enable readOnly for pills without buttons

## SUMMARY:
Enable readOnly for pills when buttons are present
## GITHUB ISSUE (Open Source Contributors)

## JIRA TASK (Eightfold Employees Only):
https://eightfoldai.atlassian.net/jira/software/c/projects/ENG/boards/50?modal=detail&selectedIssue=ENG-30670
## CHANGE TYPE:


## TEST COVERAGE:


## TEST PLAN:
- go to storybook's Pill component canvas
- In default Pills and Pills with icon sections
- check that they are not hoverable and no hover effect are implemented
- switch to other sections (with buttons)
- check that they are hoverable and have hover effects
